Userland/Applications/DisplaySettings/BackgroundSettingsWidget.cpp

```diff
@@ -31,29 +30,23 @@
 
 namespace DisplaySettings {
 
-ErrorOr<NonnullRefPtr<BackgroundSettingsWidget>> BackgroundSettingsWidget::try_create(bool& background_settings_changed)
+ErrorOr<NonnullRefPtr<BackgroundSettingsWidget>> BackgroundSettingsWidget::try_create(bool* background_settings_changed)
 {
-    auto background_settings_widget = TRY(adopt_nonnull_ref_or_enomem(new (nothrow) BackgroundSettingsWidget(background_settings_changed)));
+    auto widget = TRY(BackgroundSettingsWidget::try_create());
```

Comment:
I do realize the naming is redundant, but keeping the variable name as-is makes the diff easier to look at.